### DISCUSSION TITLE
duplicated hiragana on kana layout (ja)

### DISCUSSION THREAD

**Discussion by KonstantinDjairo** [google/mozc#776] [Q&A]
when i type the key "-" it returns ほ, which is fine, but when i type '=' , it also returns ほ
the expected character should be へ

this is the only problem, and i couldn't find a way to remap that key
this is my mozc dotfiles:

[ibus_config.textproto](https://pastebin.com/T7s6mPEP)
i tried to remap at [.mozc](https://pastebin.com/LE9fNmnw), since xev returns keycode 21 for '='
<br></br>
![image](https://github.com/google/mozc/assets/53496273/ce74e516-5392-4943-a604-ddb0c5a3f4ec)

**Comment by hiroyuki-komatsu** [comment]
Hi KonstantinDjairo,

I guess you use an US layout keyboard (e.g. `Shift` + `-` is mapped to `_`), but your configuration uses it as the JP layout in `ibus_coinfg.textproto` (e.g. `Shift` + `-` is mapped to `=`).

You might want to set "us" instead of "jp" to `layout`.

```
engines {
  name : "mozc-on"
  longname : "Mozc:あ"
  layout : "jp"  # "us" for the US layout keyboard.
  layout_variant : ""
  layout_option : ""
  rank : 80
  symbol : "あ"
  composition_mode: HIRAGANA
}
```
